demos/http/common/src/presigned_urls_gen.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
```

Comment:
This file is copied from https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1318 and changes are the responsibility of that PR.